### PR TITLE
Add balance PR related rules to the CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by gbasood**
Apparently, these need some harder guidelines than general PRs.
Let me know if you think this is clear enough or needs to be changed up.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ CONTRIBUTING TO VGSTATION
 * **Large changes require discussion.**  If you're doing a large, game-changing modification, or a new layout for something, discussion with the community is required as of 26/6/2014.  Map and sprite changes require pictures of before and after.  **MAINTAINERS ARE NOT IMMUNE TO THIS.  GET YOUR ASS IN IRC.**
```
Comment (N3X15) [inline CONTRIBUTING.md]: > IRC

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ CONTRIBUTING TO VGSTATION
...
 * Merging your own PRs is considered bad practice, as it generally means you bypass peer review, which is a core part of how we develop.
 
+# Balance changes
+ * **Balance changes must require that feedback be sought out from the players.** For example, server polls. This does not mean you pop into #code-talk in Discord and ask about it once. Good examples include:
+   * Server polls
+   * bringing attention to your PR via in-game OOC, ideally over several time slots.
```
Comment (Intigracy) [inline CONTRIBUTING.md]: muh capitalization

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -7,9 +7,16 @@ CONTRIBUTING TO VGSTATION
...
+* **Large changes require discussion.**  If you're doing a large, game-changing modification, or a new layout for something, discussion with the community is required as of 26/6/2014.  Map and sprite changes require pictures of before and after.  **MAINTAINERS ARE NOT IMMUNE TO THIS.  GET YOUR ASS IN THE CODE DISCORD.** (link in README.md)
 * Merging your own PRs is considered bad practice, as it generally means you bypass peer review, which is a core part of how we develop.
 
+# Balance changes
+ * **Balance changes must require that feedback be sought out from the players.** For example, server polls. This does not mean you pop into #code-talk in Discord and ask about it once. Good examples include:
+   * Server polls. Duration of the poll should be longer than 24 hours at minimum. Use best judgment.
+   * Bringing attention to your PR via in-game OOC, ideally over several time slots.
+ * **Balance change PRs need changelogs. Always.**
+ * **If you are a collaborator,** allow sufficient time for feedback to be gathered, and make sure that it *has* been gathered.
```
Comment (Intigracy) [inline CONTRIBUTING.md:18]: How about something about not begging for merges before the other requirements are met?

Comment (gbasood) [reply CONTRIBUTING.md:18]: I feel like people doing that would have no read this in the first place.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -7,9 +7,16 @@ CONTRIBUTING TO VGSTATION
  * This means, primarily, that you shouldn't fix bugs **and** add content in the same PR. When we mean 'bundling', we mean making one PR for multiple, unrelated changes.
 * **Test your changes.**  PRs that do not compile will not be accepted.
  * Testing your changes locally is incredibly important. If you break the serb we will be very upset with you.
-* **Large changes require discussion.**  If you're doing a large, game-changing modification, or a new layout for something, discussion with the community is required as of 26/6/2014.  Map and sprite changes require pictures of before and after.  **MAINTAINERS ARE NOT IMMUNE TO THIS.  GET YOUR ASS IN IRC.**
+* **Large changes require discussion.**  If you're doing a large, game-changing modification, or a new layout for something, discussion with the community is required as of 26/6/2014.  Map and sprite changes require pictures of before and after.  **MAINTAINERS ARE NOT IMMUNE TO THIS.  GET YOUR ASS IN THE CODE DISCORD.** (link in README.md)
```
Comment (N3X15) [inline CONTRIBUTING.md:10]: Needs minimum discussion duration

Comment (gbasood) [reply CONTRIBUTING.md:10]: I'm not sure how you would quantify that. Suggestions?

Comment (Intigracy) [reply CONTRIBUTING.md:10]: I think the minimum discussion duration is already handled by the 24h before merge portion.

**Comment by N3X15** [comment]
I have no further objections.

**Comment by N3X15** [comment]
Bumped importance, added to balance tag since it's currently blocking Balance PRs.

**Pull request merged**